config/config.exs

```diff
@@ -53,6 +53,8 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :trento, Trento.Commanded, adapter: Trento.Commanded
```

Comment:
As always, the hardest is to find an appropriate name XD

Reply:
We could bake our own "command dispatcher" in the future, which will follow an extended behaviour (dispatch_many, dispatch_retry) and so on, so as for now the naming is fine I think.